scripts/check-pr.sh

```diff
@@ -49,7 +49,7 @@ function check_diff {
   local line
   local entry
 
-  git_diff=$(git diff --name-status --find-copies-harder --diff-filter=AC --relative=pages/ master)
+  git_diff=$(git diff --name-status --find-copies-harder --diff-filter=AC --relative=pages/ main)
```

Comment:
:exclamation: This edition needs change the default branch on this repository.

Reply:
You are right, good catch. I will change that back.